infoschema/perfschema/const.go

```diff
@@ -385,72 +385,72 @@ const tableStagesHistoryLong = "CREATE TABLE if not exists performance_schema."
 	"NESTING_EVENT_TYPE		ENUM('TRANSACTION','STATEMENT','STAGE'));"
 
 // Fields in `events_statements_summary_by_digest` and `events_statements_summary_by_digest_history` are the same.
-const fieldsInEventsStatementsSummary = "SUMMARY_BEGIN_TIME TIMESTAMP(6) NOT NULL," +
-	"SUMMARY_END_TIME TIMESTAMP(6) NOT NULL," +
-	"STMT_TYPE VARCHAR(64) NOT NULL," +
-	"SCHEMA_NAME VARCHAR(64) DEFAULT NULL," +
+const fieldsInEventsStatementsSummary = "SUMMARY_BEGIN_TIME TIMESTAMP(6) NOT NULL COMMENT 'Begin time of this summary'," +
```

Comment:
should this comments be the same as MySQL's?

Reply:
There's no comment in MySQL. And we have much more fields than that in MySQL, which must be explained.

Reply:
sounds ok